core/src/main/java/greencity/exception/handler/CustomExceptionHandler.java

```diff
@@ -558,4 +562,14 @@ public final ResponseEntity<Object> handleResourceNotFoundException(ResourceNotF
 
         return ResponseEntity.status(HttpStatus.NOT_FOUND).body(exceptionResponse);
     }
+
+    @Override
+    protected ResponseEntity<Object> handleHttpRequestMethodNotSupported(
+        HttpRequestMethodNotSupportedException ex, HttpHeaders headers, HttpStatusCode status, WebRequest request) {
+        ResponseEntity<Object> response = EndpointValidationHelper.response(ex, headers, request);
+        if (response == null) {
+            return super.handleHttpRequestMethodNotSupported(ex, headers, status, request);
+        }
+        return EndpointValidationHelper.response(ex, headers, request);
```

Comment:
any reason to call `EndpointValidationHelper.response(ex, headers, request)` twice?
Why cant we return the `response` instead?